### DISCUSSION TITLE
Table row click issue when using scoped slots

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14877] [General - Components / Directives / etc] [answered]
Dear Quasar Team,

I did a demo to illustrate this issue, could you please have a look?
https://codepen.io/Stanley-549393092/pen/yLEbzqG?editors=1011
As you can see, when double click or drag cell (copy cell content), the click event is also triggered. Can we stop that?

Regards

**Comment by pdanpdan** [comment, marked as answer]
This is something you have to take care of - the dblclick event comes after the click event
https://codepen.io/pdanpdan/pen/BaVRqop?editors=1011

**Reply by 549393092** [reply]
@pdanpdan Thanks a lot, this is a good trick!
